.scripts/ApiCompat.ps1

```diff
@@ -0,0 +1,43 @@
+
+param(
+	[Parameter(Mandatory = $true)] [String]$BuildConfiguration
+)
+
+Write-Host "BuildConfiguration" $BuildConfiguration
+
+$apiTool="tool\.store\microsoft.dotnet.apicompat.tool\7.0.100\microsoft.dotnet.apicompat.tool\7.0.100\tools\net6.0\any\Microsoft.DotNet.ApiCompat.Tool.dll"        
+$apiToolRightFolder="Output"
+
+[xml]$xfiles = Get-Content .\ApiCompat.Files.xml
+$e = [char]0x274C
```

Comment:
Fyi, github emojis:
https://gist.github.com/rxaviers/7360908

You can use markdown instead of hardcoded unicode chars
:white_check_mark:
:x: